### PR TITLE
fix: update footer links

### PR DESCRIPTION
**Pull request by vasco-santos**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -43,19 +43,19 @@ function Footer () {
         <div className='md:w-1/4'>
           <h2 className='font-semibold pb-2 pt-16 md:pt-0'>Resources</h2>
           <nav>
-            <a className='block py-2 hover:text-blue-400' href="/quickstart">Quickstart guide</a>
+            <a className='block py-2 hover:text-blue-400' href="/docs/quickstart">Quickstart guide</a>
             <a className='block py-2 hover:text-blue-400' href="/faq">FAQ</a>
```
Comment (vasco-santos) [inline src/app/layout.tsx:47]: we don't have FAQ

File: src/app/layout.tsx [hunk 2]
```diff
@@ -43,19 +43,19 @@ function Footer () {
...
             <a className='block py-2 hover:text-blue-400' href="/contact">Contact us</a>
```
Comment (vasco-santos) [inline src/app/layout.tsx:48]: We don;t have contact page

Comment (vasco-santos) [reply src/app/layout.tsx]: added email

File: src/app/layout.tsx (hunk 1 above)
Comment (vasco-santos) [reply src/app/layout.tsx]: we will add markdown file in docs

**Review by olizilla — APPROVED** [review]

**Pull request merged**